content/docs/details/how-roadie-connects/index.md

```diff
@@ -0,0 +1,143 @@
+
+Regardless of the setup, Roadie can also securely connect to software that runs inside the customer’s network or their cloud account.
+
+### Option 1: Internet Public APIs
+
+Some bespoke customer APIs are exposed on the public internet. If this is the case, the API functions just like a SaaS solution, and the methods discussed above (Oauth2 and proxying) will work.
+
+If it is not possible to expose the bespoke API to the entire public internet, it may be possible to open it up to Roadie IP addresses only. That way, connections can only be made from a specific small set of originating IPs. Our docs contain the [list of originating IPs and hostnames we use](/docs/details/allowlisting-roadie-traffic/). You can likely work with your IT team to allow traffic from these origins.
+
+### Option 2: The broker
+
+For APIs which are only available on the customer’s internal network, we use a broker service to connect securely.
+
+The broker is a Node.js service that you run inside your infrastructure to provide a secure tunnel for Roadie traffic. It was originally created by def-first security company, **[Snyk](https://docs.snyk.io/features/snyk-broker/broker-introduction)**. The code is open-source. We are actively using it with existing customers for Kubernetes API access and other Backstage and Roadie plugins. You don’t need to be a Snyk user to use the broker.
+
+![An architecture diagram from the Snyk docs. It shows how the broker is used to securely connect to an on-prem GitHub or Jira instance.](./broker-arch.webp)
+*Image credit: snyk.com*
+
+The benefits of the broker include:
+
+- You can allow list what Roadie can access using a config file. By default, Roadie has no access to any internal APIs.
+- Any tokens for the internal endpoints stay in the customer infrastructure. They are not shared with Roadie.
+- The broker maintains an audit log of what we access.
+- The connection is established outbound from the customer side. We cannot re-establish the connection on our own if the customer kills it.
+
+You can read more about the [broker in the Snyk docs](https://docs.snyk.io/snyk-admin/snyk-broker). Roadie also publishes specific documentation for [configuring and using the broker](/docs/integrations/broker/).
+
+### Option 3: AWS Roles
+
+The [Backstage Kubernetes plugin](/docs/integrations/kubernetes/) can help teams answer questions like “how many pods of my service are running in each cluster and are they healthy?”.
+
+![A widget showing a kubernetes cluster called dev0. It has one pod of the sample-service running. The pod has no errors.](./backstage-k8s-plugin.png)
+
+It can use a cross-account AWS role to securely connect to Elastic Kubernetes (EKS) clusters. The cross-account AWS role is the approach recommended by AWS for this type of access
```

Comment:
```suggestion
It can use a cross-account AWS role to securely connect to Elastic Kubernetes Service (EKS) clusters. The cross-account AWS role is the approach recommended by AWS for this type of access
```

Otherwise the abbreviation in parenthesis doesn't make sense